besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -185,6 +189,8 @@
   private final BesuPluginContextImpl besuPluginContext;
   private final StorageServiceImpl storageService;
   private final Map<String, String> environment;
+  private final MetricCategoryRegistryImpl metricCategoryRegistry =
```

Comment:
nit; could this reference be the interface rather than the concrete implementation?

Reply:
It's not in this case, as we're referencing a retrieval method that we chose not to define in the interface.

Reply:
👍